src/internal/polyfills/Array.from.js

```diff
@@ -0,0 +1,24 @@
+import isIterable from '../../isIterable';
+import isUndefined from '../../isUndefined';
+
+const copyArray = (items, mapFn, thisArg) => {
+  if (isUndefined(mapFn)) return [...items];
+  if (isUndefined(thisArg)) return [...items].map(mapFn);
+  return [...items].map(mapFn).bind(thisArg);
+};
+
+const fromArray = (items, mapFn = undefined, thisArg = undefined) => {
+  if (items == null) {
```

Comment:
This logic is incorrect. The original [polyfill](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from) makes assumption that if the `mapFn` is provided, it must be a function. We're assuming following: If `items` is `nil` we're throwing error about that fact that second argument must be a function. It should be:

```js
if (isNotNil(mapFn) && isNotFunction(mapFn) {
  throw new TypeError(
     'Array.from: when provided, the second argument must be a function'
  );
}

Reply:
I hope I understand it correctly. The second argument check was added